### DISCUSSION TITLE
Can't Join CS50

### DISCUSSION THREAD

**Discussion by jmordoj** [community/community#64757] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I’m trying to join CS50x 2023 (the Edx/Harvard Course)

To do so I must follow the invite:
https://submit.cs50.io/invites/9770b67479384c4d8c37790779e466d9

but when I select "Join Course" I get:
**failed to give pull permissions to repo [404] b'{"message":"Not Found","documentation_url":"https://docs.github.com/rest/teams/teams#add-or-update-team-repository-permissions"}'**

I am a total newbie, so I am out of my depths here.
Could somebody please point me in the right direction.

*I did search for an answer here and found nothing

Thanks

**Comment by arafajs** [comment]
The error message you are getting indicates that the GitHub API could not find the team or repository that you are trying to add permissions to. This could be due to a number of reasons, such as:

- The team or repository does not exist.
- You do not have permission to add permissions to the team or repository.
- There is a problem with the GitHub API.

To troubleshoot this issue, you can try the following steps:

1. Make sure that the team and repository that you are trying to add permissions to actually exist. You can do this by visiting the GitHub repository page for the team or repository.
2. Check your permissions to see if you have permission to add permissions to the team or repository. You can do this by going to your GitHub profile and clicking on the "Permissions" tab.

If you are still having problems, you can contact GitHub support for help.

**Comment by jmordoj** [comment]
Thanks for your help

**Reply by arafajs** [reply]
Your welcome. @jmordoj 
Please, Check the answer as "The Answer" if it helps you.

**Reply by spennacchione** [reply]
I have the same problem and still can't solve it, could you help me?

**Comment by AIbagger** [comment]
I have the same problem and i cant solve ir. please @arafajs give a more detailed explanation on how to solve this problem

**Reply by Rulin-7** [reply]
Hi there did you manage to resolve it? I have the same issue and don't really understand the answer :<
